ignite/base/events_driven.py

```diff
@@ -257,34 +262,35 @@ class EventsDrivenState:
     Args:
         engine: ignite engine :class:`~ignite.base.events_driven.EventsDriven` that used to access
             the allowed events and their counters.
-        event_to_attr: mapping consists of the events from :class:`~ignite.engine.events.Events`
-            or any other custom events added by :meth:`~ignite.base.events_driven.EventsDriven.register_events`.
+        attr_to_events: mapping consists of the attributes mapped to a list events from
+            :class:`~ignite.engine.events.Events` or any other custom events added
+            by :meth:`~ignite.base.events_driven.EventsDriven.register_events`.
```

Comment:
Here user do not understand which exactly event is used with the attribute.

Reply:
I put a note to clarify this part
> Note:
        The attributes will be retrieved according to the first element in the list of the event at `attr_to_events`.

Reply:
You should say about this note here, then. Otherwise, it wont be read.